configuration/pih/htmlforms/mentalHealth.xml

```diff
@@ -383,6 +385,41 @@
                     <obs id="phq9Score" conceptId="CIEL:165137"/>
                 </span>
             </p>
+            </span>
+
+            <span class="list-inline">
+            <p>
+                <label>
+                    <uimessage code="pihcore.ncd.mental.gad7Score"/> (0-21):
+                </label>
+                <span class="small">
+                    <obs id="gad7Score" conceptId="CIEL:165185"/>
+                </span>
+            </p>
+            </span>
+
+            <span class="list-inline">
+                <p>
+                    <label>
+                        <uimessage code="WHODAS Score"/> (0-4):
+                    </label>
+                    <span class="small">
+                        <obs id="WHODASScore" conceptId="CIEL:163226"/>
+                    </span>
+                </p>
+            </span>
+
+            <span class="list-inline">
+                <p>
+                    <label>
+                        <uimessage code="CGI Score"/> (0-4):
```

Comment:
@tommyiversonj There are 3 CGI scores (CGI-I, CGI-E, CGI-S).  For other countries, we include all 3 of the scores.  See CIEL:163221.  And the individual scores are 0-7.

You are using CGI-I in your code which ranges from 0-7.

Reply:
I changed the four to seven